UsingGoogleSignInButton.md

```diff
@@ -0,0 +1,44 @@
+# Using the Google Sign In Button
```

Comment:
How should we link this content to README.md?  Alternatively, would it be better to merge it into README.md?

Reply:
I will add it to the `README`.